### ISSUE TITLE
Display architecture - increase test coverage

### ISSUE THREAD

**Issue by JakubVanek** [ev3dev-lang-java/ev3dev-lang-java#791]
Current unit testing solution for the display stack is inadequate. A revision of the current testing code is necessary.

**Comment by jabrena** [comment]
Yep, it is something missing in the library. When we added the nice EV3 Display support, we forget adding it.

Currently, we have this status:

![image](https://user-images.githubusercontent.com/147263/104104570-3eb60900-52a9-11eb-9dea-c63c0fa37518.png)

![image](https://user-images.githubusercontent.com/147263/104104581-57262380-52a9-11eb-916f-a18146e1237e.png)

- LCDStretch 
- LCD
- LCDStretch.Flusher

![image](https://user-images.githubusercontent.com/147263/104104617-8b99df80-52a9-11eb-9894-fcbd94dba4bf.png)

![image](https://user-images.githubusercontent.com/147263/104104625-9c4a5580-52a9-11eb-9015-65bab48ce88d.png)

With that contribution, we could enable again the following configuration to reach 80% of code coverage:

```groovy
jacocoTestCoverageVerification {
    violationRules {
        rule {
            limit {
                //minimum = 0.75 // implement more tests
                minimum = 0.2
            }
        }
    }
}
```

**Comment by JakubVanek** [comment]
Given that LCDJessie is implemented as a separate class, it seems that the BitFramebuffer & FramebufferProvider paths are mostly redundant. Removing them might reduce the amount of code that needs testing.

**Comment by jabrena** [comment]
Create a PR with this small change. I will merge directly to master.
